### DISCUSSION TITLE
How to use multer with native Nuxt 3... without express?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#15976] [Questions] [answered]
How to use multer with native Nuxt 3... without express?
I know that multer is a library for express
but maybe u know analogs for nuxt 3?
[link](https://www.bacancytechnology.com/blog/file-upload-using-multer-with-nodejs-and-express)

**Comment by mihail727** [comment, marked as answer]
my solution [here](https://github.com/nuxt/framework/discussions/4051)

**Reply by cyperdark** [reply]
doesnt work on release version
